### ISSUE TITLE
DNAJB9 protein level in serum elevated

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10209]
**Preferred term label:**
DNAJB9 protein level in serum elevated

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5762944/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Fibrillary Glomerulonephritis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000001
- label: Elevated circulating DNAJB9 protein concentration
- parents: HP:0010876
- pmid: PMID:29340314
- def: Concentration of DnaJ homolog subfamily B member 9 (DNAJB9) in the blood circulation above the upper limit of normal.
- comment: DNAJB9 is a proteomic tissue biomarker for fibrillary glomerulonephritis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10209
- synonyms: DNAJB9 protein level in serum elevated(NONE)|Elevated circulating DnaJ homolog subfamily B member 9 (DNAJB9) concentration(NONE)
